python-varlink.spec

```diff
@@ -1,3 +1,7 @@
+%if 0%{?fedora} || 0%{?rhel} >= 8
```

Comment:
Thank you! EPEL 7 contains Python 3.6, so the python3 package can be built there as well.

Reply:
I don't see python3 setuptools for RHEL7/Centos7,  so I'm building only python2 version for them. 
Even before including python2 version to EPEL, we need to build it for Fedora and then include to EPEL fedora-wishlist.

Reply:
python36-setuptools is part of EPEL: https://koji.fedoraproject.org/koji/packageinfo?packageID=21685

What do you mean with the EPEL fedora wishlist?

Reply:
Can we enable EPEL during building package for RHEL in Copr?

About wishlist -  https://fedoraproject.org/wiki/Archive:EPEL_Package_Wish_List?rd=EPEL_Package_Wish_List
Is there another way how to get python-varlink in EPEL?

Reply:
@haraldh is the maintainer for python-varlink in Fedora, so you need to ask him to maintain it for EPEL as well. The EPEL wish list is not used anymore (it is in the archive namespace to indicate this).

Reply:
@tyll Yes, this is what the issue #12 is about. After this PR will be merged @haraldh need only toggle a new checkbox for EPEL7 in Copr.